content/en/security/application_security/guide/manage_account_theft_appsec.md

```diff
@@ -0,0 +1,674 @@
+
+Successful logins should be considered suspicious.
+
+[1]: https://app.datadoghq.com/security?query=%40workflow.rule.type%3A"Application%20Security"%20category%3Aaccount_takeover&product=appsec
+
+{{% /tab %}}
+
+{{% tab "Credential Stuffing" %}}
+
+This signal flagged a lot of activity coming from a few IPs and is closely related to its distributed variant. You might need to use the distributed credential stuffing method.
+
+Start by extracting a list of suspicious IPs from the signal side panel
+
+<!-- screenshot -->
+
+From the list of IPs, you can craft a [Traces][25] query to review all the activity from suspected IPs. Follow this template:
+
+`@appsec.security_activity:business_logic.users.login.* @http.client_ip:(<IPs>)`
+
+Successful logins should be considered suspicious.
+
+{{% /tab %}}
+
+{{% tab "Distributed Credential Stuffing" %}}
+
+This signal flagged a large increase in login failures in one service. If the attack is large enough, this signal might trigger both Bruteforce or Credential Stuffing signals. The signal is also able to detect diffuse attacks.
+
+In the diffuse attacks case, attacker attributes are available in the signal.
+
+<!-- screenshot -->
+
+1. Click **Investigate in full screen**.   
+2. In **Attacker Attributes**, select the cluster and click, then, in **Traces**, click **View in ASM Protection Trace Explorer**.
+
+This gets you to the trace explorer with filters set to the flagged attributes. You can start the investigation with the current query, but you should expand it to also match login successes on top of the failures. Review the exhaustiveness/accuracy of the filter using the technique described above (in the paragraph before the table).
+
+<!-- sceenshot -->
+
+In the case those attributes are inaccurate/incomplete, you may try to identify further traits to isolate the attacker activity. The most useful traits are:
+
+1. User agent: `@http.user_agent`  
+2. ASN: `@http.client_ip_details.as.domain`  
+3. Threat intelligence: `@threat_intel.results.category`  
+4. URL: `@http.url`  
+5. Fingerprint, when available: `@appsec.fingerpring.*`
+
+<!-- sceenshot -->
+
+You may use Top List or Timeseries to identify the traits whose distribution most closely matches the attack.
+
+You may need multiple sets of filters, each possibly including multiple traits. Behind the scenes, the attacker may be using multiple randomized templates. This work identifies the constants in those templates.
+
+{{% /tab %}}
+{{< /tabs >}}
+
+#### Review login successes and failures
+
+Reviewing login successes and failures helps to identify the following:
+
+* What the attackers are after so that you can block them.  
+* What the attackers are doing so that you can catch them, even if they change their scripts.   
+* How successful the attackers are so that you can take back the accounts where they took control and see how much time you have to react.
```

Comment:
```suggestion
* How successful the attackers are so that you can take back the accounts they took control of and see how much time you have to react.
```